frontend/models/product-list/index.ts

```diff
@@ -227,6 +242,31 @@ type ApiProductList = NonNullable<
    >[0]['attributes']
 >;
 
+function convertAncestorsToStrapiFormat(
+   ancestors: any
+): ApiProductList['parent'] | null {
+   const ancestor = ancestors.shift();
+   if (ancestor == null) {
+      return null;
+   }
+   if (ancestor.wikiid === 11000) {
+      ancestor['type'] = 'all_parts';
+      ancestor['title'] = 'All';
+      ancestor['handle'] = 'Parts';
+   }
+   return {
+      data: {
+         attributes: {
+            type: ancestor.type,
+            title: ancestor.title + ' Parts',
+            handle: ancestor.handle ?? '',
```

Comment:
What is this handle?

Reply:
the ancestors but for most of them that information isn't available to be created from the ifixit info. But its a required attribute by ts rn